aws/table_aws_cloudcontrol_resource.go

```diff
@@ -18,7 +18,7 @@ func tableAwsCloudControlResource(_ context.Context) *plugin.Table {
 		List: &plugin.ListConfig{
 			KeyColumns: []*plugin.KeyColumn{
 				{Name: "type_name", Require: plugin.Required},
-				{Name: "resource_model", Require: plugin.Optional},
+				{Name: "resource_model", Require: plugin.Optional, CacheMatch: "exact"},
```

Comment:
```suggestion
				{Name: "resource_model", Require: plugin.Optional},
```

If we don't really need to do exact cache matching, we should remove